go.sum

```diff
@@ -123,6 +123,8 @@ github.com/dgraph-io/badger v1.6.0 h1:DshxFxZWXUcO0xX476VJC07Xsr6ZCBVRHKZ93Oh7Ev
 github.com/dgraph-io/badger v1.6.0/go.mod h1:zwt7syl517jmP8s94KqSxTlM6IMsdhYy6psNgSztDR4=
 github.com/dgraph-io/badger/v3 v3.0.0-20210303071538-42d5e9510a4d h1:6ykqhVkgHypRkqIU95iFL3Ir+sY4bt9+gkElBu7PPQ0=
 github.com/dgraph-io/badger/v3 v3.0.0-20210303071538-42d5e9510a4d/go.mod h1:m0D1mIoSHk/Ph/5nUU1Ft3raZHHQ3M7VBs5Iu1qbdHw=
+github.com/dgraph-io/badger/v3 v3.0.0-20210303185901-6121fdc57546 h1:MpsltqvTNzssvU1j9TX4pxTOeGEpxE2BDNbEzrJBE6I=
```

Comment:
`go mod tidy`

Reply:
i have to change these once other prs gets merged. will do that